extensions/notification/NotificationConfigManager.js

```diff
@@ -1,19 +1,73 @@
 const joi = require('joi');
 
+const { ZenkoMetrics } = require('arsenal').metrics;
 const LRUCache = require('arsenal').algorithms
     .cache.LRUCache;
 const MongoClient = require('mongodb').MongoClient;
-
 const constants = require('./constants');
 
 const paramsJoi = joi.object({
     mongoConfig: joi.object().required(),
+    origin: joi.string().required(),
     logger: joi.object().required(),
 }).required();
 
 const MAX_CACHED_ENTRIES = Number(process.env.MAX_CACHED_BUCKET_NOTIFICATION_CONFIGS)
     || 1000;
 
+// Config manager is used in both the QueuePopulator and QueueProcessor
+// we use this label to distinguish between the two
+const CONFIG_MANAGER_LABEL_ORIGIN =  'origin';
+// should equal true if config manager's cache was hit during a get operation
+const CONFIG_MANAGER_CACHE_HIT = 'cache_hit';
+
+const cacheUpdates = ZenkoMetrics.createCounter({
+    name: 'notification_config_manager_cache_updates',
+    help: 'Total number of cache updates',
+    labelNames: [
+        CONFIG_MANAGER_LABEL_ORIGIN,
+    ],
+});
+
+const configGet = ZenkoMetrics.createCounter({
+    name: 'notification_config_manager_config_get',
+    help: 'Total number of config get operations in the notification config manager',
+    labelNames: [
+        CONFIG_MANAGER_LABEL_ORIGIN,
+        CONFIG_MANAGER_CACHE_HIT,
+    ],
+});
+
+const cachedBuckets = ZenkoMetrics.createGauge({
+    name: 'notification_config_manager_cached_buckets_count',
+    help: 'Total number of cached buckets in the notification config manager',
+    labelNames: [
+        CONFIG_MANAGER_LABEL_ORIGIN,
+    ],
+});
+
+function onConfigManagerCacheUpdate(origin, op) {
+    cacheUpdates.inc({
```

Comment:
the op shoud probably be added as a label, to better see the 'repartitions' of updates